assets/src/blocks/Submenu/submenuFunctions.js

```diff
@@ -1,3 +1,5 @@
+import md5 from 'md5';
```

Comment:
We actually don't need to use md5 for this, not sure why it was done like that. Since we have access to all heading titles we could just use the title itself and suffix it with a number if we encounter it twice.

Reply:
it was might be because of special char in header like `'` (apostrophe). right now if the header has apostrophe the submenu link wont work. (eg string "it's new to world" )